rig/machine_control/machine_controller.py

```diff
@@ -917,6 +935,30 @@ def send_signal(self, signal, app_id=Required):
         self._send_scp(0, 0, 0, SCPCommands.signal, arg1, arg2, arg3)
 
     @ContextMixin.use_contextual_arguments
+    def app_start(self, app_id=Required):
+        """Transmit a signal to start the application."""
+        self.send_signal(consts.AppSignal.start, app_id)
+
+    @ContextMixin.use_contextual_arguments
+    def app_stop(self, app_id=Required):
+        """Transmit a signal to stop the application."""
+        self.send_signal(consts.AppSignal.stop, app_id)
+
+    @ContextMixin.use_contextual_arguments
+    def sync0(self, app_id=Required):
```

Comment:
As in `count_sync0`? I'm not sure I see why that would be particularly useful as separate to this?